mantidproject/mantid
# Removed old docs from techniques folder

Description of work.
This PR removes old docs from the techniques folder for scripts that were replaced by the new ISIS Powder scripts.

**To test:**
Ensure build passes

Fixes N/A

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
